.github/workflows/lint.yml

```
@@ -0,0 +1,50 @@
+name: Lint
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+    env:
+      NPM_VERSION: "8.3.0"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
```

can use nvmrc file in frontend

```suggestion
          node-version-file: frontend/.nvmrc
```

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file